README.md

```diff
@@ -1,77 +1,122 @@
-# 🎸 Bandcamp Data Pipeline
+# 🎼 Apollo
 
 
 
 ## 🔎 Overview
 
 ### 📝 Description
-> Welcome to Team Apollo's project revolving around the creation of a data pipeline via the use of Bandcamp's API and webscraping. The data is then used to create a dashboard which will provide various information on artists and genres regarding their sales and popularity both recently and over time.
+> Welcome to Team Apollo's sales tracker revolving around the creation of an ETL pipeline via the use of Bandcamp's API and webscraping. The data is then used to create three different outputs which will provide various information on artists and genres regarding their sales and popularity both recently and over time. These are: **dashboard**, **email notifications**, and **PDFs**. This results in our **tracker** which is provided to consumers as a service.
 
 ### 👩‍💼 Stakeholder Requirements
-- Obtain latest sales data from Bandcamp on artists/genre
-- Automatically alert, notify, and/or send PDFs to subscribers regarding this data.
+- Obtain **latest** sales data from Bandcamp on artists/genre
+- Automatically **alert**, **notify**, and/or **send** PDFs to subscribers regarding this data.
 
 ### 🎯 Deliverables
-- A full data pipeline, hosted in the cloud.
-- A long-term storage solution for the data extracted from Bandcamp.
-- Some form of visualisation of the data.
+- A **full data pipeline**, hosted in the cloud.
+- A **long-term storage solution** for the data extracted from Bandcamp.
+- Some form of **visualisation of the data**.
```

Comment:
Think a bit too casual? Maybe just say sales data visualization

Reply:
Fine, I've updated it.